src/plugin.js

```diff
@@ -1,61 +1,84 @@
 'use strict'
 
 const { createWriteStream } = require('fs')
+const { readFile } = require('fs/promises')
```

Comment:
This is probably not a problem anymore nowadays, but I remember Rollup not detecting `fs/promises` as a built-in. So I'd just check the bundled code to make sure that this line remains unchanged.

Reply:
Looks good.